resources/core/charts/service-catalog/charts/catalog/values.yaml

```diff
@@ -80,6 +80,32 @@ apiserver:
       useEmbedded: false
       # etcd URL(s); override this if NOT using embedded etcd
       servers: https://core-service-catalog-etcd-client.kyma-system.svc.cluster.local:2379
+      # etcd image to use
```

Comment:
if we are not using embedded etcd, does it make sense to specify those values here? It can be a misleading a little bit. On the other hand, it is easier to update service-catalog chart; but in such case, we should add a short comment that those values refers only to embedded etcd.  If we want to keep this values, we should probably copy etcd-pvc.yaml from the original chart.

Reply:
I agree with you. IMO the more important is to have the easy update of the service-catalog chart because we will do that quite often. So I kept those values and also I added `etcd-pvc.yaml`.  Comment also added.